### PR TITLE
Coding style - PHP code tag and control structures

### PR DESCRIPTION
**Pull request by marcochirienti**
### Summary of Changes

Removed unnecessary white-spaces after the PHP code opening tag.			
	Added one white-space before the PHP code closing tag where missing.			
	Fixed style for control structures alternative syntax.  

### Testing Instructions

Code review.

### Documentation Changes Required

None.

### CODE REVIEW

**Comment by shur** [comment]
I have tested this item :white_check_mark: successfully on e2c70ee94d57bdc6f74b0faf69aa74deafb7ea3e<br /><br />Tested successfully by code review.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/13141">issues.joomla.org/tracker/joomla-cms/13141</a>.</sub>

File: components/com_contact/views/contact/tmpl/default_links.php [hunk 1]
```diff
@@ -16,7 +16,7 @@
 <?php if ($this->params->get('presentation_style') === 'tabs') : ?>
 	<?php echo JHtml::_('bootstrap.addTab', 'myTab', 'display-links', JText::_('COM_CONTACT_LINKS')); ?>
 <?php endif; ?>
-<?php if ($this->params->get('presentation_style') === 'plain'):?>
+<?php if ($this->params->get('presentation_style') === 'plain') : ?>
 	<?php echo '<h3>' . JText::_('COM_CONTACT_LINKS') . '</h3>';  ?>
```
Comment (shur) [inline components/com_contact/views/contact/tmpl/default_links.php]: @infograf768 good find!

@marcochirienti please leave just one space in line20 before closing `  ?>`

**Comment by marcochirienti** [comment]
Hawk sight. Fixed.
Thanks @infograf768

**Comment by jeckodevelopment** [comment]
@zero-24 can you please review it?

**Comment by shur** [comment]
I have tested this item :white_check_mark: successfully on a23caf560304075d5c4fd4abbf34f9f7701e7fd8<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/13141">issues.joomla.org/tracker/joomla-cms/13141</a>.</sub>

**Review by zero-24 — APPROVED** [review]

**Comment by zero-24** [comment]
Looks good from a quick review

**Comment by jeckodevelopment** [comment]
I have tested this item :white_check_mark: successfully on a23caf560304075d5c4fd4abbf34f9f7701e7fd8<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/13141">issues.joomla.org/tracker/joomla-cms/13141</a>.</sub>

**Comment by jeckodevelopment** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/13141">issues.joomla.org/tracker/joomla-cms/13141</a>.</sub>

**Pull request merged**